src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/MongoDbSessionHandlerTest.php

```diff
@@ -167,7 +167,7 @@ public function testWrite()
             $that->assertEquals('bar', $data[$that->options['data_field']]->getData());
             $that->assertInstanceOf('MongoDB\BSON\UTCDateTime', $data[$that->options['time_field']]);
             $that->assertInstanceOf('MongoDB\BSON\UTCDateTime', $data[$that->options['expiry_field']]);
-            $that->assertGreaterThanOrEqual($expectedExpiry, round(intval((string) $data[$that->options['expiry_field']]) / 1000));
+            $that->assertGreaterThanOrEqual($expectedExpiry, round((int) ((string) $data[$that->options['expiry_field']]) / 1000));
```

Comment:
does the ``(int)`` casting apply before or after the division by 1000 here ? This code is less readable than before, as it requires knowing the operator precedence (and I cannot guarantee that it works as before here without checking in the php documentation, as I don't know this precedence by heart)